Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -52,8 +54,21 @@ def run_installer(pkg_description)
         ]
         args << "-verboseR" if Hbc.verbose
         args << "-allowUntrusted" if pkg_install_opts :allow_untrusted
+        if pkg_install_opts :choices
+          args << "-applyChoiceChangesXML"
+          args << choices_xml
+        end
         @command.run!("/usr/sbin/installer", sudo: true, args: args, print_stdout: true)
       end
+
+      def choices_xml
+        path = @cask.staged_path.join("Choices.xml")
```

Comment:
Could you change this to use a [Tempfile](https://github.com/Homebrew/brew/blob/8d15bddedb2f5796193f79a9a97cd6271367e251/Library/Homebrew/cask/lib/hbc/container/dmg.rb#L67-L77)? Since this is generated from the cask file, there is no need to keep this around and clutter up the `staged_path` with it.